src/python/WMComponent/WorkQueueManager/WorkQueueManagerWorkPoller.py

```diff
@@ -66,6 +70,8 @@ def pullWork(self):
         try:
             if self.retrieveCondition():
                 work = self.queue.pullWork()
+            else:
+                self.queue.logger.info("Retrieve condition is not true: won't pull work")
```

Comment:
How about we say `Workqueue didn't pass the retrieve condition: NOT pulling work`. 
I was going to say we could make it a warning log level to catch more attention, but maybe it's not a good fit in here... (?)